dockerfiles/Dockerfile.archlinux

```diff
@@ -1,23 +1,19 @@
-FROM archlinux/base
+FROM scratch
 MAINTAINER Iterait a.s. <docker@iterait.com>
 
+COPY --from=archlinux/base / /
+# add cleanup hook
+ADD 99-cleanup.hook /usr/share/libalpm/hooks/
+
 # install common packages
 RUN pacman --noconfirm --needed -Syu \
-      git \
-      base-devel \
-      cmake \
-      gtkglext \
-      hdf5 \
-      gtk3 \
-      opencv \
-      qt5-base \
-      python \
-      vim \
-      python-numpy \
-      python-pip \
-      sudo \
-      boost \
-      openssh
+	binutils \
+	fakeroot \
+        git \
+        sudo \
+    && rm -rf \
+        /README \
+        /etc/pacman.d/mirrorlist.pacnew
```

Comment:
Are those two present in the `archlinux/base` image?

Reply:
I guess we could remove `README`, the mirrorlist might show up as a result of the system update.

Reply:
ok, let's remove README. Shouldn't we put the `mirrorlist.pacnew` to the cleanup hook, then?

Reply:
I guess. Or we can just let it live its life, it's just a few kBs.